frontend/ui/widget/sortwidget.lua

```diff
@@ -407,7 +353,7 @@ function SortWidget:nextPage()
 end
 
 function SortWidget:prevPage()
-    local new_page = math.max(self.show_page-1, 1)
+    local new_page = math.max(self.show_page - 1, 1)
```

Comment:
Ditto above for `SortWidget:nextPage()`.

Reply:
https://github.com/koreader/koreader/pull/7544#discussion_r612638902

Reply:
Well, I tend to prefer spaces no matter the context, but regardless of what you end up with, it should be consistent ;).

Right now one has spaces and not the other ;).

Reply:
Done, I like spaces.

Reply:
It's not the spaces, it's the diff polluting.

Reply:
Restored originals
https://github.com/koreader/koreader/blob/7092df7f4e09f595239afee0706ce20609abefd1/frontend/ui/widget/sortwidget.lua#L399
https://github.com/koreader/koreader/blob/7092df7f4e09f595239afee0706ce20609abefd1/frontend/ui/widget/sortwidget.lua#L410

Reply:
Thanks. :-)